modules/vebal/vebal-voting-list.service.ts

```diff
@@ -114,14 +114,14 @@ export class VeBalVotingListService {
         return gaugesWithStaking;
     }
 
-    async syncVotingGauges() {
+    public async syncVotingGauges() {
         const onchainGaugeAddresses = await this.votingGauges.getVotingGaugeAddresses();
 
         return this.sync(onchainGaugeAddresses);
     }
 
     async sync(votingGaugeAddresses: string[]) {
-        const chunks = chunk(votingGaugeAddresses, 100);
+        const chunks = chunk(votingGaugeAddresses, 50);
```

Comment:
@franzns I found that the new `gaugeControllerHelperAbi` was failing when doing a multicall with 100 elements. This fixes the problem but we could do something better. Could you help me with this?

Reply:
think it's totally fine like this